siriuspy/siriuspy/pwrsupply/psctrl/pscwriters.py

```diff
@@ -411,6 +411,19 @@ def execute(self, value=None):
             self.pru_controller.sofb_update_variables_state()
 
 
+class ParamUpdate(Function):
+    """ParamUpdate Function."""
+
+    def __init__(self, device_ids, pru_controller):
+        """Create command to update parameters."""
+        self.device_ids = device_ids
```

Comment:
maybe use `self._devices_ids` instead. And change `class Voltage` that is not standardized either.